recipes/argo-cd.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: argo
+    url: https://argoproj.github.io/argo-helm
+    type: helm
+  name: argo-cd
+  namespace: "argocd"
+  sha256:
+  version: 
```

Comment:
Could you also set a version, this would be a chart version in the helm chart case.

Reply:
I Will do that! but is there any way to use the latest helm chart version always from the URL? by setting the value of version to either `latest` or keeping it blank?